source/material_model/rheology/elasticity.cc

```diff
@@ -371,7 +371,7 @@ namespace aspect
                                          const SymmetricTensor<2,dim> &stress,
                                          const double shear_modulus) const
       {
-        const SymmetricTensor<2,dim> edot = 2. * (deviator(strain_rate)) + stress /
+        const SymmetricTensor<2,dim> edot = deviator(strain_rate) + 0.5*stress /
```

Comment:
I'm actually surprised this did not alter more of the VEP tests. At first glance, the change above combined with the one in `visco_plastic`  (remove `current_edot_ii /= 2.`) should not give equivalent values, as you first take the square root of the second invariant (line 377 below). I must be missing something here :-)

Reply:
@bobmyhill - Never mind, I just saw your updated documentation in the comment above. Reading through that now.

Reply:
> All the terms in the second invariant involve multiplying two elements of the initial tensor, so that should mean that
>    <img src="https://render.githubusercontent.com/render/math?math=\dot{\varepsilon} (\text{new}) = \sqrt{|s_{\text{II}} / 4|} = \sqrt{|s_{\text{II}}|}/2 = \dot{\varepsilon}">

Well that's convenient 😄